config/config.yaml

```diff
@@ -2,10 +2,10 @@
 # The configuration of key.yaml has a higher priority and will not enter git
 
 #### if OpenAI
-
-#OPENAI_API_KEY: "YOUR_API_KEY"
-#OPENAI_API_BASE: "YOUR_API_BASE"
+## You can get OPENAI_API_BASE url from https://github.com/beidongjiedeguang/openai-forward
+OPENAI_API_BASE: "https://api.openai-forward.com/v1"
```

Comment:
Is it better to add `#` here?

Reply:
Give user a default available value, so that user only needs to modify OPENAI_API_KEY and OPENAI_API_MODEL.